Vagrantfile

```diff
@@ -18,12 +18,12 @@ module KatelloDeploy
       :libvirt    => 'http://m0dlx.com/files/foreman/boxes/centos64.box'
     },
     :centos7 => {
-      :box_name   => 'centos7',
+      :box_name   => 'centos7_1',
```

Comment:
We labeled the centos6 box as just '6', but I guess we have to do this for the new physical box?